src/Service/RequestService.php

```diff
@@ -1036,8 +1036,8 @@ public function requestHandler(array $data, array $configuration): Response
                 $code = Response::HTTP_BAD_REQUEST;
             }
 
-            // If we have a response return that
-            if ($event->getData()['response']) {
+            // If we have a response return that.
+            if (isset($event->getData()['response']) === true && empty($event->getData()['response']) === false) {
```

Comment:
do we have a case where event->getData()['response'] is not set?

Reply:
Yes the zgw to xxllnc sync synchronizes on the background and you do not want to return anything from there to the requesting user.